packages/block-editor/src/components/use-block-commands/index.js

```diff
@@ -189,7 +190,9 @@ const useActionsCommands = () => {
 		);
 	} );
 	const canRemove = canRemoveBlocks( clientIds, rootClientId );
-	const canMove = canMoveBlocks( clientIds, rootClientId );
+	const canMove =
+		canMoveBlocks( clientIds, rootClientId ) &&
+		getBlockCount( rootClientId ) !== 1;
```

Comment:
Should this be `getBlockCount( rootClientId ) > 1` as `0 !== 1` is also true?

Reply:
Hm.. this command already needs some selected client ids so I don't see how we could have `0`.. Am I missing something here?

Reply:
You're right. Ignore my pre-coffee comment 🙇